### PR TITLE
Padding & Colours

### PR DESCRIPTION
**Pull request by victor-ponce**
Agregamos las fuentes a ser utilizadas, así como colores y algunos tamaños del mismo.
I added the font families to be used, as well as colors and some sizes of it.

### CODE REVIEW

**Comment by victor-ponce** [comment]
Hi Erick, Good Night

I made some changes to my polishing branch like the padding. colours and some improvements in my code. My last Commit is 546e8fe45e named as Padding & Colours and i saw that the vercel deployment are on the right menu on deployments section , preview polishing. here is the link: [https://mentorship-portfolio-cq02gh77b-victor-ponce.vercel.app/](url)

Sorry i'm just learning how to use GitHub, thanks

File: css/styles.css [hunk 1]
```diff
@@ -57,45 +91,76 @@ td {
...
 }
+
 /************* PROFESSIONAL EXPERIENCE******************/
 .accordion {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 400;
     padding: 50px 0;
+
```
Comment (eKoh951) [inline css/styles.css]: Remove line break

File: css/styles.css [hunk 2]
```diff
@@ -57,45 +91,76 @@ td {
...
     background-color: #262626;
-    color: #ffffff;
+    color: #e5e5e5;
     text-align: left;
 }
+
 .accordion-button {
     color: #5c6bcf;
-    font-weight: bold;
+    font-weight: 700;
```
Comment (eKoh951) [inline css/styles.css:127]: Remove line break

File: css/styles.css [hunk 3]
```diff
@@ -57,45 +91,76 @@ td {
...
 
 }
+
 .accordion-item {
```
Comment (eKoh951) [inline css/styles.css:130]: Add line break 1 line before

File: css/styles.css [hunk 4]
```diff
@@ -57,45 +91,76 @@ td {
...
     border: 1px solid #606060;
-   
+
 }
+
 .accordion-button:not(.collapsed) {
```
Comment (eKoh951) [inline css/styles.css:135]: Add line break 1 line before

File: css/styles.css [hunk 5]
```diff
@@ -57,45 +91,76 @@ td {
...
     color: inherit;
     background: #cfd3f1;
     color: #5c6bcf
 }
+
 .collapsed {
```
Comment (eKoh951) [inline css/styles.css:141]: Add line break 1 line before

File: css/styles.css (hunk 2 above)
Comment (eKoh951) [inline css/styles.css]: Remove line break

File: css/styles.css [hunk 6]
```diff
@@ -57,45 +91,76 @@ td {
...
     color: #5c6bcf;
-    font-weight: bold;
+    font-weight: 700;
 
 }
+
 .accordion-item {
     border: 1px solid #606060;
-   
+
```
Comment (eKoh951) [inline css/styles.css]: Remove line break

File: css/styles.css [hunk 7]
```diff
@@ -124,12 +191,14 @@ td {
...
     transform: translate(-50%, -50%) scale(0);
-    font-size: 1.6rem;
-    color: black;
+    font-size: 1.25rem;
+    font-weight: 400;
+    color: #212529;
     transition: .2s ease;
     opacity: 0;
-  
+
```
Comment (eKoh951) [inline css/styles.css]: Remove line break

File: css/styles.css [hunk 8]
```diff
@@ -139,30 +208,81 @@ td {
...
 .level {
+    color: #e5e5e5;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 400;
+    margin: 4px 0;
+    padding: 8px 16px;
+}
+.course {
```
Comment (eKoh951) [inline css/styles.css]: Add line break

File: css/styles.css [hunk 9]
```diff
@@ -17,24 +26,49 @@ h1 {
...
+    border: 1px solid #5c6bcf;
 }
+
+.btn-outline-light:hover {
+    background: #8895EA;
+    border: 1px solid #5c6bcf;
+    color: #5c6bcf;
+}
+
 .description {
```
Comment (eKoh951) [inline css/styles.css]: Add a more relevant class name, what is this `description` for?

File: css/styles.css [hunk 10]
```diff
@@ -17,24 +26,49 @@ h1 {
...
+    font-style: normal;
+    font-weight: 400;
     padding-bottom: 1%;
 }
+
 .navbar {
     padding-bottom: 5%;
 }
+
 .row {
```
Comment (eKoh951) [inline css/styles.css:70]: Add a more relevant class name

File: css/styles.css [hunk 11]
```diff
@@ -139,30 +208,81 @@ td {
...
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+}
+
 
 /***************** EDUCATION ******************/
 
 .education {
```
Comment (eKoh951) [inline css/styles.css:221]: What is this for? education text, container, card? Add a more relevant class name

File: css/styles.css [hunk 12]
```diff
@@ -139,30 +208,81 @@ td {
...
     color: #d6d7d9;
 }
+
 .data-scroll {
     border-left: 1px solid #ffffff;
-  padding: 1% 5%;
+    padding: 1% 5%;
 }
+
 .level {
```
Comment (eKoh951) [inline css/styles.css]: Add a more relevant class name

File: css/styles.css [hunk 13]
```diff
@@ -139,30 +208,81 @@ td {
...
+.data-scroll {
+    color: #d6d7d9;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 400;    
+}
+/*.active a {
+    color: #5c6bcf;
+}
```
Comment (eKoh951) [inline css/styles.css]: Remove unused code

File: css/styles.css [hunk 14]
```diff
@@ -171,13 +291,23 @@ td {
...
     color: #262626;
+    font-family: 'Open Sans', sans-serif;
+    font-size: .875rem;
+    font-style: normal;
+    font-weight: 600;
 }
+
 .social {
     border-radius: 100%;
 }
```
Comment (eKoh951) [inline css/styles.css]: Add the `.social` class css attributes in the same line (190)

File: css/styles.css [hunk 15]
```diff
@@ -188,14 +318,17 @@ hr {
...
-        
-    }
-    .footer-link {
-        color: #c0c1c2;
-    }
-    .footer-row {
-        padding: 10px 0;
-    }
\ No newline at end of file
+
```
Comment (eKoh951) [inline css/styles.css]: Remove line break

File: css/styles.css [hunk 16]
```diff
@@ -188,14 +318,17 @@ hr {
     margin-top: 0;
     margin-bottom: 0;
 }
+
 .names {
```
Comment (eKoh951) [inline css/styles.css:284]: Add a more relevant name, what is this `.names` class for?

**Review by eKoh951 — COMMENTED** [review]
Overall looks good.
Just the spacing here is a little bit too large:
image

**Pull request merged**